### ISSUE TITLE
(Pricing table) Per sit limits listed or mentioned? Github?

### ISSUE THREAD

**Issue by laralfield** [bwdinc/asada#3]
Hey Chase! What are your thoughts on how many websites should be supported per license, and how should we present that to customers in the pricing table? I think if included, it should go in the pricing table at the very top. 

I _really_ like your price differentiation with the first option that limits the support period to 60 days. This makes a lot of sense from a customer fit perspective for the hobbyists set. I also think that making the add-ons free for a year as a part of the kitchen sink option makes a lot of sense. I like the use of the word add-ons, and we can update any documentation to reflect this. 

Thanks!

**Comment by ChaseWiseman** [comment]
I was going back and forth on the "x sites supported" item as well and left it out for the initial commit.

The question I had was: how is that applied? There is no core licensing as of 1.3.3, so how does the support process differ for supporting a single site vs. 3 vs. unlimited? Best I can come up with is the user entering their domain(s) as part of the support request.

**Comment by laralfield** [comment]
Yes, I think the domain should just be a part of the support request, and enforceability will fall to our team members on Zendesk handling incoming support requests, and matching them up with our paid queue, or our queue for all other incoming requests. 

So with https://easydigitaldownloads.com/support/pricing/, I'm curious if the support is for an unlimited number of domains, and if the support itself is capped. I don't think it is, and I've read some of the frustration and almost extremes Pippin has gone to in order to serve his support queue because he's such a dedicated developer. I want to stay far away from any of these types of very draining practices  down the road, hence the Tech Coin introduction which I got some great feedback on at WordCamp ME. But if we really think that unlimited is the way to go, then we can consider that, too. Let's just be sure to compare it against as many successful WordPress plugin companies using a model similar to ours as we can, first.

Thanks!! So exciting!

**Comment by ChaseWiseman** [comment]
I'd rather not go unlimited either. The tricky part is figuring out how to limit it! Even fully paid plugins with a site limit don't _really_ limit support. The site limit is only for automatic updates, as the customer can just switch the licensed domain whenever they want support on a different site.

Yes, it looks like EDD's support is forum-based. Essentially your paying to have your questions looked into/answered ahead of the basic support forums.

As you mentioned, we could certainly go to the Tech Coins and do away with the yearly subscription!
